.eslintignore

```diff
@@ -6,9 +6,9 @@ packages/*/build
 **/*.js
 **/*.esnext
 **/*.mjs
-**/test/fixtures/**/default-types/*.ts
+**/tests/fixtures/**/default-types/*.ts
 
 # Don't lint fixture graphql files as they include schemas
-packages/graphql-config-utilities/src/tests/fixtures/**/*.graphql
-packages/graphql-typescript-definitions/test/fixtures/**/*.graphql
-packages/graphql-validate-fixtures/test/fixtures/**/*.graphql
+packages/graphql-config-utilities/src/testss/fixtures/**/*.graphql
```

Comment:
Double s here might be the cause of the linting failures

Reply:
Thank you. This was driving me nuts.